core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreManager.java

```diff
@@ -2784,52 +2787,66 @@ protected String preparePassword(String password, String saltValue) throws UserS
     }
 
     /**
-     * Prepare the password including the salt, and hashes if hash algorithm is provided
+     * Prepare the password including the salt, and hashes if hash algorithm is provided.
      *
-     * @param password original password value
-     * @param saltValue salt value
-     * @return  hashed password or plain text password as a String
-     * @throws UserStoreException
+     * @param password  Original password value which needs to be hashed
+     * @param saltValue Salt value
+     * @return Hashed password or plain text password as a String
+     * @throws UserStoreException The exception thrown at hashing the passwords.
      */
```

Comment:
Shall we fix the formatting here? WDYT?

Reply:
I am not seeing any formatting errors here other than full stops.

Reply:
Then let's get it fixed